SpatialGDK/Source/Private/EngineClasses/SpatialNetDriver.cpp

```diff
@@ -133,13 +134,24 @@ void USpatialNetDriver::OnMapLoaded(UWorld* LoadedWorld)
 		}
 	}
 
+	// If we have hit OnMapLoaded and we already have a connection then we know we are in ServerTravel.
+	// This involves some clean up ready for a fresh instance.
+	// For the server (with authority over the GSM) it also involves loading a fresh snapshot and toggling Accepting players when ready.
+	if (Connection->IsConnected())
+	{
+		UE_LOG(LogSpatialOSNetDriver, Warning, TEXT("Loaded Map %s. Server in ServerTravel. Cleaning up old connection..."), *LoadedWorld->GetName());
+		OnConnected();
```

Comment:
I feel like the name `OnConnected` implies it's only used as a callback. How about `ResetConnection`?

Reply:
After much discussion we went with `OnMapLoadedAndConnected`